src/ToolBox/SOS/Strike/strike.cpp

```diff
@@ -6878,9 +6878,43 @@ int CNotification::s_condemnedGen = -1;
 
 BOOL CheckCLRNotificationEvent(DEBUG_LAST_EVENT_INFO_EXCEPTION* pdle)
 {
+    ISOSDacInterface4 *psos4 = NULL;
+    CLRDATA_ADDRESS arguments[3];
+    HRESULT Status;
+
+    if (SUCCEEDED(Status = g_sos->QueryInterface(__uuidof(ISOSDacInterface4), (void**) &psos4)))
+    {
+        int count = _countof(arguments);
+        int countNeeded = 0;
+
+        Status = psos4->GetClrNotification(arguments, count, &countNeeded);
+        psos4->Release();
+
+        if (SUCCEEDED(Status))
```

Comment:
I forgot that S_FALSE means succeeded (hence the S_ instead of E_).  I'll get fix that.